index.js

```diff
@@ -170,7 +170,22 @@ var faces = [
   "乙(のっの)乙",
   "ヾ(¯∇￣๑)",
   "\_(ʘ_ʘ)_/",
-  "༼;´༎ຶ ۝ ༎ຶ༽"
+  "༼;´༎ຶ ۝ ༎ຶ༽",
+  "(oo,)",
+  "(xx,)",
+  "(^^,)",
+  "(**,)",
+  "(--,)",
+  "<(oo.<)",
+  "(``,)",
+  "(uu,)",
+  "(@@,)",
+  "(!!,)",
+  "(\",)",
+  "(,,.)",
+  "(*,*)",
```

Comment:
http://memesvault.com/wp-content/uploads/What-Meme-09.jpg
and
http://www.smilequick.com/wp-content/uploads/photo-gallery/smile-1.jpg

;)